### DISCUSSION TITLE
Multiple external scanners

### DISCUSSION THREAD

**Discussion by lbahtarliev** [rspamd/rspamd#4609] [Q&A]
Hello community,

I am using rspamd for mail checking against ICAP enabled solutions. I am pretty sure i`ve read somewhere that multiple external scanners definitions are allowed. Indeed, they are allowed and this configuration is accepted by the daemon. But it scans the email only against the first defined ICAP service in the external_scanners module. 
Thus my question - If we have multiple external scanners definitions, is it possible to try them all when scanning an attachment/mail? And maybe stop at first detection?

Thanks,
Lyuben

**Comment by fatalbanana** [comment]
Multiple ICAP scanners work indeed as far as I see.

This comment from the documentation is relevant:
~~~
# multiple scanners could be checked, for each we create a configuration block with an arbitrary name
icap {
~~~

Furthermore `symbol` setting of these scanners should be distinct.
